xbmc/guilib/GUIWindow.cpp

```diff
@@ -1038,7 +1038,7 @@ void CGUIWindow::SetProperty(const std::string &strKey, const CVariant &value)
 
 CVariant CGUIWindow::GetProperty(const std::string &strKey) const
 {
-  CSingleLock lock(*this);
+  CSingleLock lock(const_cast<CGUIWindow&>(*this));
```

Comment:
I thought you wanted to remove the const_casts, not to introduce new ones?

Reply:
xbmc/threads/SharedSection.h and xbmc/threads/SingleLock.h are the files which casted away the constness.
For this one I could have introduced a locking variable, but I didn't want to change it in this PR.

Reply:
Ah, overlooked this in the first place. What an ugly hack. Thanks for cleaning up that mess.